string servo_state
string track_state
float64 yaw
float64 pitch
